catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/util/impl/CatalogRuntimeException.java

```diff
@@ -11,22 +11,22 @@
  * is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.commands.util;
+package ddf.catalog.util.impl;
 
-public class CatalogCommandRuntimeException extends RuntimeException {
-    public CatalogCommandRuntimeException() {
+public class CatalogRuntimeException extends RuntimeException {
+    public CatalogRuntimeException() {
```

Comment:
But what if one is merely passing an exception and no message is needed? For example the one case where this is used (https://github.com/codice/ddf/pull/2080#discussion_r123301891) There really is no message to give, the real message of failure is contained in the exception that is being wrapped.